### PR TITLE
Added initial API info. RHOARDOC-1145

### PR DESCRIPTION
**Pull request by rhoads-zach**
@lance @lholmquist thoughts?

### CODE REVIEW

File: docs/topics/proc_using-nodemon-for-dev.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
+[#using-nodemon-for-development]
+= Using nodemon for Development
+
+The {NodeJS} runtime includes link:https://nodemon.io/[nodemon^], which watches a directory and restarts your application when it sees a change. This is useful during
```
Comment (Ladicek) [inline docs/topics/proc_using-nodemon-for-dev.adoc]: during what? :-)

File: docs/topics/proc_using-nodemon-for-dev.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+[discrete]
+== Prerequisites
+* An application you want to use with nodemon. For example, xref:mission-http-api-nodejs[a booster].
+* `npm` installed.
+
+[discrete]
+== Procedure
+
+. Use `npm` to install nodemon.
```
Comment (Ladicek) [inline docs/topics/proc_using-nodemon-for-dev.adoc]: Above, you said that the Node.js runtime _includes_ nodemon. Does it really, if you need to install it?

**Comment by Ladicek** [comment]
So, I don't know anything about `nodemon`, but it seemed useful. Why is it gone now? I'd be totally fine with the previous content, if it didn't tell that nodemon is included. (Well maybe it created an impression that it's somehow "supported" as part of RHOAR, but that's just about wordsmithing, right? :-) )

**Comment by rhoads-zach** [comment]
@Ladicek i might re-introduce in the debugging section

**Comment by rhoads-zach** [comment]
@lance @lholmquist thoughts on the the new section?

**Comment by rhoads-zach** [comment]
@lholmquist @lance @Ladicek 

Re-added nodemon as an appendix

**Comment by danbev** [comment]
> We also need to confirm with @danbev if the S2I builder image for production includes this.

`nodemon` is currently not included in the production S2I image. My take on this from the last discussion we had is that we'd document the steps to set it up but not provide anything except the node runtime itself in the docker image.

**Comment by rhoads-zach** [comment]
@lance @danbev thank you for the feedback. I'll remove the nodemon content again and revisit at a later date.

FYI @Ladicek ^^

**Comment by Ladicek** [comment]
I think the content we have right now is perfectly fine for local usage, and that's still valuable. Sure, you can have `nodemon` running in the OpenShift pod and use it by `oc rsync`-ing stuff (could you even use `nodemon` to trigger the `os rsync`?).

Anyway, I'm not insisting on keeping it right now; we can always revisit later.

**Comment by rhoads-zach** [comment]
@lance removed nodemon for now and will revisit later. can you give a +1 if you're good with the current state?

**Comment by lance** [comment]
@rhoads-zach I can, although we are still talking about this on the team and ways in which we can support it (not as official support, but just so the build doesn't break). Do you want to wait until we sort that out?

**Comment by rhoads-zach** [comment]
@lance i think we should break the nodemon stuff out into a different issue so we can wrap this PR up. Do you have any objections to that? Re-adding the content is not hard.

**Comment by rhoads-zach** [comment]
thank you @lance 

@inoxx03 @tradej feel free to merge.

**Comment by inoxx03** [comment]
@rhoads-zach LGTM, merging...

**Comment by inoxx03** [comment]
merged: c776dbdf0c079a4512cc1f55addd1979663674cc

**Pull request closed without merging**